navigation.yaml

```diff
@@ -495,171 +496,167 @@ products:
               url: https://juju.is/docs/sdk/from-zero-to-hero-write-your-first-kubernetes-charm
 
 solutions:
-  - cta: Explore our solutions
-    main_lists:
-      - title: Categories
-        primary_links:
-          - links:
-              - title: AI
-                url: /solutions/ai
-              - title: Data
-                url: /data
-              - title: Infrastructure
-                url: /solutions/infrastructure
-              - title: IoT
-                url: /PLACE-HOLDER
-              - title: Managed IT services
-                url: /PLACE-HOLDER
-              - title: Secure open source
-                url: /PLACE-HOLDER
-              - title: Ubuntu OS
-                url: /PLACE-HOLDER
+  cta: Explore our solutions
+  main_lists:
+    - title: Categories
+      link_list:
+        - title: AI
+          url: /solutions/ai
+        - title: Data
+          url: /data
+        - title: Infrastructure
+          url: /solutions/infrastructure
+        - title: IoT
+          url: /PLACE-HOLDER
+        - title: Managed IT services
+          url: /PLACE-HOLDER
+        - title: Secure open source
+          url: /PLACE-HOLDER
+        - title: Ubuntu OS
+          url: /PLACE-HOLDER
 
-      - title: Industries
-        primary_links:
-          - links:
-              - title: Automotive
-                url: https://ubuntu.com/automotive
-              - title: Finance
-                url: https://ubuntu.com/financial-services
-              - title: Industrial
-                url: https://ubuntu.com/industrial
-              - title: Government
-                url: https://ubuntu.com/gov
-              - title: Telco
-                url: /solutions/telco
+    - title: Industries
+      link_list:
+        - title: Automotive
+          url: https://ubuntu.com/automotive
+        - title: Finance
+          url: https://ubuntu.com/financial-services
+        - title: Industrial
+          url: https://ubuntu.com/industrial
+        - title: Government
+          url: https://ubuntu.com/gov
+        - title: Telco
+          url: /solutions/telco
 
-    highlighted_secondary_links:
-      - title: Case studies
-        links:
-          - description: Sicredi transforms its infrastructure strategy with open source.
-            url: https://ubuntu.com/engage/sicredi-openstack-case-study
-            image_url: /PLACE-HOLDER
-          - description: LaunchDarkly becomes the first FedRAMP-authorised feature management platform thanks to Ubuntu Pro.
-            url: /PLACE-HOLDER
-            image_url: /PLACE-HOLDER
-          - description: Canonical deploys infrastructure solutions and managed IT services for ESA's critical space mission operations
-            url: https://ubuntu.com/engage/European-space-agency-case-study
-            image_url: /PLACE-HOLDER
+  previewondary_links:
```

Comment:
There's a typo here for `preview_secondary_links` causing the case studies to not show up